gammapy/scripts/tests/test_spectrum_pipe.py

```diff
@@ -0,0 +1,45 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.tests.helper import pytest
+from astropy.tests.helper import assert_quantity_allclose
+import astropy.units as u
+from ...utils.energy import EnergyBounds
+from ...utils.testing import requires_dependency, requires_data
+from ...background.tests.test_reflected import obs_list, on_region
+from ...spectrum.models import PowerLaw
+from .. import SpectrumAnalysisIACT
+
+
+def get_config():
+    """Get test config, extend to several scenarios"""
+    test_conf = dict(
+        outdir=None,
+        background=dict(on_region=on_region()),
+        extraction=dict(),
+        fit=dict(model=PowerLaw(index=2,
```

Comment:
I find this hard to read.
Suggest to introduce local variable `model = ` above and then put it here.